jenkins_job_wrecker/modules/triggers.py

```diff
@@ -238,6 +243,14 @@ def gerrittrigger(top, parent):
             pass  # Unconfigurable Attribute
         else:
             raise NotImplementedError("Not implemented Gerrit Trigger Plugin's attribute: ", child.tag)
+
+        if child.tag in mapper_gerrit_build.get_tags():
+            # Jenkins Job Builder implementation uses "override-votes"
+            # key to override default vote values. For detail:
+            # https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.gerrit
+            is_override_votes = True
```

Comment:
Would it be ok to put this under:

    elif mapper_gerrit_build.map_element(child, gerrit_trigger):

?

Then you wouldn't need `.get_tags()` at all.

Reply:
Yes it would, updating the code.